developer-workflow/c-api.rst

```diff
@@ -122,6 +122,11 @@ To add a new set of tests (or extract a set out of the monolithic
 - Call the ``_PyTestCapi_Init_*`` from ``PyInit__testcapi`` in
   ``Modules/_testcapimodule.c``.
 
+- Add the new C file to :cpy-file:`Modules/Setup.stdlib.in`,
+  :cpy-file:`PCbuild/_testcapi.vcxproj` and
+  :cpy-file:`PCbuild/_testcapi.vcxproj.filters`,
```

Comment:
Perhaps we should mention that these are `noeol` files?

Reply:
Perhaps. But I'm not sure what that means in practice, on other people's computers...

Reply:
I guess we can just leave it like this; Git warns you if you try to add newline at the end of these files anyways (see `.gitattributes` IIRC).